tests/test_api_transactions.py

```diff
@@ -91,81 +91,87 @@ def test_create_transaction_422(client: TestClient):
 @pytest.mark.parametrize(
     "method,url", [("POST", "/transactions"), ("PATCH", "/transactions/1")]
 )
+@pytest.mark.parametrize(
+    "payload,expected_error_type",
```

Comment:
ruff (black) auto-format is so nice here, can you believe there was a time I would do this manually?! 😱